<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signupApp</class>
 <widget class="QDialog" name="signupApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>-10</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add New User details</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submitNewUser_pushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>350</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="enterName_lineEdit">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="maxLength">
       <number>25</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="enterPassword_lineEdit">
      <property name="maxLength">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Role</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="selectRole_comboBox">
      <item>
       <property name="text">
        <string>Admin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Super Admin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Service</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Doctor</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
